content/altinn-studio/getting-started/Intro-altinn-studio/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Introduksjon til Altinn Studio
+linktitle: Introduksjon
+description: Altinn Studio er vårt nye verktøy for å utvikle offentlige digitale tjenester. Dette kan være alt fra helt enkle skjema til avanserte applikasjoner. Dette er en brukerveiledning for hvordan man utvikler tjenester i Altinn ved bruk av Altinn Studio.
+weight: 10
+aliases:
+- /altinn-studio/intro-altinn-studio
+---
+
+Skal du bruke TUL eller Altinn Studio? Les [kom i gang med utvikling](/docs/kom-i-gang-med-utvikling/#skal-du-bruke-tul-eller-altinn-studio). 
+
+{{% children description="true" %}}
+
+
+Prosessen for å lage skjemaer og meldinger vil være forskjellig alt etter som hvor stor eller liten målgruppen er, hvor mye data som skal behandles eller hvor mye jobb det er å legge til rette for at egne systemer kan snakke med Altinn.
+
+Slik går du frem: 
```

Comment:
Er ikke alt dette bare unødvendig gjentagelse av https://altinn.github.io/docs/utviklingsguider/digitaldialog/ ? Gir dette noe verdi?

Reply:
Enig at det er my gjentakelser, men det er også en del endringer i forhold til 3.0. Ønsket av serviceleder når de skal presentere Altinn 3.0 for sine TE.